crypto/s2n_hash.c

```diff
@@ -95,7 +95,7 @@ int s2n_hash_is_available(s2n_hash_algorithm alg)
         is_available = 1;
         break;
     default:
```

Comment:
do we even need a default?

is it better to be explicit and require hashes to be added here and generate a compiler error if new ones are added to the enum and not to this switch?

Reply:
Nice, will do that.